frontend/packages/kubevirt-plugin/integration-tests/tests/utils/utils.ts

```diff
@@ -2,12 +2,19 @@
 import { execSync } from 'child_process';
 import * as _ from 'lodash';
 import { $, $$, ElementFinder, browser, by, ExpectedConditions as until } from 'protractor';
-import { appHost } from '../../../../../integration-tests/protractor.conf';
+import { appHost, testName } from '../../../../../integration-tests/protractor.conf';
+import { click } from '../../../../console-shared/src/test-utils/utils';
```

Comment:
Nit: we can import from `@console/shared` here.

Please go over any `../console-shared` (cross-package relative import) occurences in `kubevirt-plugin/integration-tests` and replace them with proper module import.